### PR TITLE
[JIRA](DLF-344) Minor change to participate() as per Hecham's verbal instructions

### PR DESCRIPTION
**Pull request by squishybear-zz**

### CODE REVIEW

**Comment by squishybear-zz** [comment]
@hghazal Please CR one last time before I merge and release.

File: djsixpack/tests/class_tests.py [hunk 1]
```diff
@@ -292,3 +278,34 @@ class GetBucketTest(SixpackTest):
 
         record_count = SixpackParticipant.objects.filter(experiment_name='get_bucket', unique_attr=10).count()
         self.assertEquals(record_count, 1)
+
+
+class ConvertTest(TestCase):
```
Comment (squishybear-zz) [inline djsixpack/tests/class_tests.py:282]: Restored the ConvertTest tests.

**Comment by squishybear-zz** [comment]
@adrianmaurer and @hghazal I've made some minor changes to the participate() logic and updated the tests.  Please review today if possible (there are changes in ACCT and API waiting).

**Comment by 4x4notfound** [comment]
:ship: looks good. Thanks for taking care of this.

**Pull request merged**